Shipr-Helm/templates/redis-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ Values.redis.name }}
+spec:
```

Comment:
What do you think, should redis be a deployment or a statefulset?

Reply:
Yes statefulset is best approach here

Reply:
Well, I think we should leave it to the user to decide if they want a statefulset or a deployment.